### PR TITLE
PDP: Fixes in content for team members and environments

### PR DESCRIPTION
**Pull request by zachtil**
## Describe this PR

Fix a few content issues in PDP content for environments and team members

## Changes

### CODE REVIEW

File: content/800-data-platform/250-environments/index.mdx [hunk 1]
```diff
@@ -7,21 +7,21 @@ tocDepth: 3
 
 <TopBlock>
 
-For any project in the Prisma Data Platform, you can create multiple environments. You can associate each environment with a different database, Data Proxy location, and a different branch of your GitHub repository.
+For any project in the Prisma Data Platform, you can create multiple environments. You can associate each environment with a different database, Data Proxy location, and a different branch of your GitHub repository (if schema syncrhonization from GitHub is enabled).
```
Comment (keerlu) [inline content/800-data-platform/250-environments/index.mdx]:
```suggestion
For any project in the Prisma Data Platform, you can create multiple environments. You can associate each environment with a different database, Data Proxy location, and a different branch of your GitHub repository (if schema synchronization from GitHub is enabled).
```

File: content/800-data-platform/250-environments/index.mdx [hunk 2]
```diff
@@ -7,21 +7,21 @@ tocDepth: 3
...
 
 You can dedicate each environment to a different stage of your database application life-cycle, such as production, staging, development, or any others that make sense for your project.
 
-For each environment in a [project](/data-platform/projects), you define:
+For each new environment in a [project](/data-platform/projects), you define:
 
-- a GitHub branch (based on the repository of the Production environment)
 - a database connection string
 - a Data Proxy region
+- ([if schema synchronization is enabled](/data-platform/projects/create#optional-enable-schema-synchronization)) a GitHub branch from the repository of the Production environment
```
Comment (keerlu) [inline content/800-data-platform/250-environments/index.mdx]: Does this mean you associate the environment with a branch from the same repo as the production environment? If so maybe:
```suggestion
- ([if schema synchronization is enabled](/data-platform/projects/create#optional-enable-schema-synchronization)) a GitHub branch from the same repository as the Production environment
```

**Review by keerlu — APPROVED** [review]
Looks good, couple of small things

File: content/800-data-platform/250-environments/index.mdx (hunk 2 above)
Comment (zachtil) [reply content/800-data-platform/250-environments/index.mdx]: That's correct, thanks for the edit!

File: content/800-data-platform/900-troubleshooting/100-cannot-edit-schema-file.mdx [hunk 3]
```diff
@@ -14,7 +14,7 @@ The Prisma Data Platform does not support the complete development workflow of m
 
 ## **Problem**
 
-For the purposes of consulting your Prisma schema, you can use **Schema Viewer** tab to check your models and their fields and definitions, but you cannot edit and develop your Prisma schema in the Prisma Data Platform.
+For the purposes of consulting your Prisma schema, you can use **Schema Viewer** to check your models and their fields and definitions, but you cannot edit and develop your Prisma schema in the Prisma Data Platform.
```
Comment (zachtil) [inline content/800-data-platform/900-troubleshooting/100-cannot-edit-schema-file.mdx]:
```suggestion
For the purposes of consulting your Prisma schema, you can use the **Schema Viewer** to check your models and their fields and definitions, but you cannot edit and develop your Prisma schema in the Prisma Data Platform.
```

**Pull request merged**